Update ens avatars and history events sections (#6107)
rotki/rotki: docs/usage_guide.rst

ENS names resolution
=====================

> Rotki automatically resolves ens name and ens avatar for every evm address that has ens name set in ethereum mainnet. If there is a primary ens name specified for an address, this name will be shown instead of a raw blockchain address across the app. You can find the blockchain address by hovering the mouse over the ens name. If an ens avatar is set, it will be shown instead of the default address icon.

Reviewer: ```suggestion
Rotki automatically resolves ens name and ens avatar for every evm address that has ens name set in ethereum mainnet. If there is a primary ens name specified for an address, this name will be shown instead of a raw blockchain address across the app. You can find the blockchain address by hovering the mouse over the ens name. If an ens avatar is set, it will be shown instead of the address blockie.
```

   :align: center

History events
=====================

> Rotki is capable of getting and decoding (understand what is happening) your events (ethereum transactions, optimism transactions, ethereum staking, etc.). When you visit the ``History Events`` section the process to obtain all the information will start. You will be able to check the status in an informative breakdown per blockchain address. Free users are limited to the 100 latest events.

Reviewer: ```suggestion
Rotki is capable of pulling and decoding a bunch of different events, ranging from EVM chain transactions to exchanges events and more. When you visit the ``History Events`` section the process to obtain all the information will start. You will be able to check the status in an informative breakdown per blockchain address. Free users are limited to a number of latest events.
```

Let's not specify here as we would need to change each time we change it.

.. image:: images/events_query_process.png
   :alt: History events query status breakdown
   :align: center

> It is possible that you need to redecode some events. To redecode an evm transaction you have two possibilities. The first of them is to click on the three dots to display the options for an EVM transaction and click on ``Redecode events``. This will start the process to read the transaction's events again and try to understand what happened in them.

Reviewer 2: you can only redecode transactions, the other events (non evm events) can't be redecode

Reviewer: ```suggestion
It is possible that you need to redecode events for an evm transaction. To do that you have two options. The first of them is to click on the three dots to display the options for an EVM transaction and click on ``Redecode events``. This will start the process to read the transaction's events again and try to understand what happened in them.
```

And what's the second option?

Author: > And what's the second option?

```
The second option is to redecode all the transactions in one page or all the transactions that have been queried. To do so you need to click on ``REDECODE EVENTS`` at the top of the page. This will allow to select between ``Only this page`` and ``All events``.
```

   :align: center

> History events can be filtered by account and by:

Reviewer: These filters can only happen in premium.
